people_m.rb

```diff
@@ -37,14 +39,41 @@ def create_person
     when '1'
       new_student = create_student
       @people << new_student
+      save_people_to_json
       puts 'New person (Student) created successfully'
     when '2'
       new_teacher = create_teacher
       @people << new_teacher
+      save_people_to_json
       puts 'New person (Teacher) created successfully'
     else
       puts 'Please enter a valid number (1 or 2)'
       create_person
     end
   end
+
+  def save_people_to_json
+    data = @people.map(&:to_h)
+    File.write('people.json', JSON.generate(data))
+  end
```

Comment:
- [ ] Great work implementing this feature. Currently, the data is saved to the _**JSON**_ file during the running of the application. The requirements state that data should be saved to the _**JSON**_ file on exit from your application. Kindly configure your code in such a way that the _**methods**_ that save the data to the _**JSON**_ files will be called on exit of the application. Please implement this for the methods that save people, books and rentals to the _**JSON**_ files. 😎 💻 

![image](https://github.com/blueberry1312/school-library/assets/77106962/a4c64bd3-4456-46da-8dd3-9cfbfebf4a81)

Reply:
Oh yes We missed that, thanks!